packaging/os/apt.py

```diff
@@ -73,6 +73,12 @@
     required: false
     default: "no"
     choices: [ "yes", "no" ]
+  allow_unauthenticated:
+    description:
+      - Ignore if packages can't be authenticated. This is useful for packages that bootstrap their own apt-key setup.
```

Comment:
@resmo what the "This is useful for packages that bootstrap their own apt-key setup" is supposed to mean from *my* PoV (me not being the submitter) is that you need `--allow-unauthenticated` for *bootstrapping* the apt-key setup, like in `apt-get install --allow-unauthenticated foobar-archive-keyring` where `foobar-archive-keyring` provides the apt-key(s).